script/install/system_preferences

```diff
@@ -14,11 +57,15 @@ install_system_preferences () {
   # System Preferences > Keyboard >
   defaults write NSGlobalDomain KeyRepeat -int 2
 
+  # System Preferences > Keyboard > Modifier Keys > Set Caps Lock to Control
+  setup_caps_lock_as_left_control
+
   # System Preferences > Keyboard >
   defaults write NSGlobalDomain InitialKeyRepeat -int 15
 
   # System Preferences > Trackpad > Tap to click
   defaults write com.apple.driver.AppleBluetoothMultitouch.trackpad Clicking -bool true
+  defaults write com.apple.AppleMultitouchTrackpad Clicking -bool true
```

Comment:
I'd prefer if we left the default, but I know I will probably get outvoted.

Reply:
Nah, that's a good point. I'd like to support customizations per user on things like this. I mean, I think you're crazy, but I'd like to support everyone's individual craziness.

Off the top of my head I don't have any ideas I love for ways to customize this per user, but we'll think of something.

Reply:
I have some bad ideas! Maybe we could turn them into good ones?